ideascube/tests/test_settings.py

```diff
@@ -14,12 +14,6 @@ def test_all_settings(request):
         module = path.purebasename
         module = '.conf.%s' % module
 
-        try:
-            settings = importlib.import_module(module, package="ideascube")
-
-        except Exception as e:
-            print(path)
-            print(module)
-            pytest.fail(str(e))
+        settings = importlib.import_module(setting_module, package="ideascube")
```

Comment:
Why not try/catch the exception anymore ?

There is no `setting_module` here. On latter commit yes, but not yet.